SPECS/kernel-mshv/50_mariner_mshv.cfg

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+        
+GRUB_CMDLINE_LINUX="$GRUB_CMDLINE_LINUX audit=0"
```

Comment:
Style question: Do we want to put these all on individual lines, or just all on one? Obviously it doesn't affect the final result just curious about style here.

Reply:
I'm fine with either way. More lines = clearer diffs, but that is about it.